file/staticFile/main.go

```diff
@@ -17,13 +17,26 @@
 package main
 
 import (
+	"context"
+
+	"github.com/cloudwego/hertz/pkg/app"
 	"github.com/cloudwego/hertz/pkg/app/server"
+	"github.com/cloudwego/hertz/pkg/protocol/consts"
 )
 
 func main() {
 	h := server.Default(server.WithHostPorts("127.0.0.1:8080"))
 
+	h.Static("/static", "./")
+
 	h.StaticFile("/main", "./main.go")
 
+	h.StaticFS("/static1", &app.FS{
```

Comment:
![image](https://user-images.githubusercontent.com/97824201/229416001-1706f40d-6614-493a-afc2-cbc54726f397.png)


It seems that only two parameters are used